Clarify the necessity of completing a batch (#2533)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/mailbox-migration/perform-G-Suite-migration.md

17. In the details pane, select the preferred option to complete the batch and click **Save**.

    The batch status will then be **Completed**.
	

> During completion, another incremental sync is run to copy any changes that have been made to the Google Workspace mailbox. Additionally, during completion, the forwarding address that routes mail from Microsoft 365 or Office 365 to Google Workspace is removed, and a forwarding address that routes mail from Google Workspace to Microsoft 365 or Office 365 is added. This ensures that any  messages received by migrated users at their Google Workspace mailboxes will be sent to their new Microsoft 365 or Office 365 address. Similarly, if any user who has not yet been migrated receives a message at their Microsoft 365 or Office 365 address, the message will get routed to their Google Workspace mailbox.

Reviewer: ```suggestion
During completion, another incremental sync is run to copy any changes that have been made to the Google Workspace mailbox. Additionally, during completion, the forwarding address that routes mail from Microsoft 365 or Office 365 to Google Workspace is removed, and a forwarding address that routes mail from Google Workspace to Microsoft 365 or Office 365 is added. This ensures that any messages received by migrated users at their Google Workspace mailboxes will be sent to their new Microsoft 365 or Office 365 address. Similarly, if any user who has not yet been migrated receives a message at their Microsoft 365 or Office 365 address, the message will get routed to their Google Workspace mailbox.
```

Author: Fine with me, that spacing is actually coming from the legacy EAC section.

    ![start the batch](../media/gsuite-mig-17-eac-start.png)

11. After the batch status changes from **Syncing** to **Synced**, you need to complete the batch. The batch status will then be **Completed**.

    ![batch syncing](../media/gsuite-mig-18-eac-syncing.png)
    
    > [!NOTE]
    > When the batch starts, all the users to be migrated will be converted from MailUsers to Mailboxes. The Microsoft 365 or Office 365 Exchange license must be assigned only after this moment. You have 30 days to assign the license.

> During completion, another incremental sync is run to copy any changes that have been made to the Google Workspace mailbox. Additionally, during completion, the forwarding address that routes mail from Microsoft 365 or Office 365 to Google Workspace is removed, and a forwarding address that routes mail from Google Workspace to Microsoft 365 or Office 365 is added. This ensures that any  messages received by migrated users at their Google Workspace mailboxes will be sent to their new Microsoft 365 or Office 365 address. Similarly, if any user who has not yet been migrated receives a message at their Microsoft 365 or Office 365 address, the message will get routed to their Google Workspace mailbox.

Reviewer: ```suggestion
During completion, another incremental sync is run to copy any changes that have been made to the Google Workspace mailbox. Additionally, during completion, the forwarding address that routes mail from Microsoft 365 or Office 365 to Google Workspace is removed, and a forwarding address that routes mail from Google Workspace to Microsoft 365 or Office 365 is added. This ensures that any messages received by migrated users at their Google Workspace mailboxes will be sent to their new Microsoft 365 or Office 365 address. Similarly, if any user who has not yet been migrated receives a message at their Microsoft 365 or Office 365 address, the message will get routed to their Google Workspace mailbox.
```

### Complete the migration batch in Microsoft 365 or Office 365

When the migration batch has reached the state of **Synced**, it needs to be completed by running the `Complete-MigrationBatch` cmdlet.

> During completion, another incremental sync is run to copy any changes that have been made to the Google Workspace mailbox. Additionally, the forwarding address that routes mail from O365 to Google Workspace is removed, and a forwarding address that routes mail from Google Workspace to O365 is added.

Reviewer: ```suggestion
During completion, another incremental sync is run to copy any changes that have been made to the Google Workspace mailbox. Additionally, the forwarding address that routes mail from Office 365 to Google Workspace is removed, and a forwarding address that routes mail from Google Workspace to Office 365 is added.
```

Author: Thanks, that's currently part of the documentation. Wouldn't it even be better to use Microsoft 365 instead since Microsoft is trying to get rid of all the Office 365 branding?

Reviewer: True, but Office 365 still exists. I realize that this is part of the current documentation, but O365 is definitely a no-no, especially since Office 365 has been used throughout the rest of the topic. I am not sure if the paragraph refers specifically to Office 365, or if it can be changed to Microsoft 365, so I would hesitate to change it. Thanks for your input, though.

Author: It is interesting that no one picked this mishap up except now, when I made a verbatim copy of the legacy section ;-)
Thanks!
